README.md

```diff
@@ -1,8 +1,17 @@
-# Beard Template Engine
+# Beard: A Blazing-Fast Template Engine
 
 [![Build Status](https://travis-ci.org/zalando/beard.svg)](https://travis-ci.org/zalando/beard)
 
-Blazing fast, [open source](https://github.com/zalando/beard), logic-less template engine written in Scala, used to parse templates using a simple syntax:
+Beard is an open-source, logic-less templating engine, written in Scala and inspired by [Mustache](https://mustache.github.io/). You can use it out-of-the-box; it has no dependencies (see Requirements list below). 
+
+What makes Beard powerful:
+
+  - Streaming. As soon as you need to render something, you can stream it to the browser. This provides high user-perceived performance.
+  - Speed. We've benchmarked it against other template engines for the JVM, and Beard performed twice as fast in terms of rendering time. (We invite you to run your own benchmarks to see if you get the same results.) It also uses ANTRL to make template compilation fast.
```

Comment:
I wouldn't say twice as fast, as this is hard to prove in every case... I'd say `faster than other template engines`